packages/core/src/observables/participant.ts

```diff
@@ -165,20 +166,34 @@ export function createIsSpeakingObserver(participant: Participant) {
   );
 }
 
-export function connectedParticipantsObserver(room: Room) {
+type ConnectedParticipantsObserverOptions = {
+  extraRoomEvents?: RoomEvent[];
```

Comment:
In my thinking the events aren't "extra", but rather the actual one's we update on. 
E.g. they default to `all` room events and if you specify them, that default is overridden with whatever you pipe in.

Reply:
maybe that would lead to `connectedParticipantsObserver` to become `participantsObserver` and the events always include the three main one's?

Reply:
What you describe sounds like we have three main events that we always want to include, even if the user decides to override the default (all events). 
If we have a base and some additional (user-defined) events, I find my naming pretty descriptive. Don't you think?